templates/helm/values.yaml.tpl

```diff
@@ -121,3 +121,11 @@ serviceAccount:
   name: {{ .ServiceAccountName }}
   annotations: {}
     # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+
+# leader election configurations
```

Comment:
Please use proper casing for this - this is documentation. Maybe you could link to the K8s documentation about leader election, in case people wonder what it is

Reply:
There is no official documentation on this topic.. it's more a controller-runtime thing. Maybe the Go docs?

Reply:
I did find this section about leader election on the K8s page for leases - https://kubernetes.io/docs/concepts/architecture/leases/#leader-election

But it's vague.